package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "to-do-list",
   "version": "1.0.0",
-  "description": "",
+  "description": "This project is about simple to do list app using webpack.",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
- [ OPTIONAL] Please it would be better if you add the start script here so that the application can easily be run using the `npm start` command. I know this should have been done already in the previous milestones. :pray:

Reply:
- As pointed out by the CR here, it will be better to have a start script similar to this 👇🏽
![Screenshot from 2023-03-04 14-09-53](https://user-images.githubusercontent.com/44624138/222903694-08f4a3d0-c31e-4542-aec4-e78c66ad94c0.png).